test/e2e/operator.go

```diff
@@ -448,9 +454,9 @@ var _ = Describe("ARO Operator - Azure Subnet Reconciler", func() {
 		for subnet, correctNSG := range subnetsToReconcile {
 			By(fmt.Sprintf("waiting for the subnet %q to be reconciled so it includes the original cluster NSG", subnet))
 			Eventually(func(g Gomega, ctx context.Context) {
-				s, err := clients.Subnet.Get(ctx, resourceGroup, vnetName, subnet, "")
+				s, err := clients.Subnet.Get(ctx, resourceGroup, vnetName, subnet, nil)
 				g.Expect(err).NotTo(HaveOccurred())
```

Comment:
Subnet field not extracted from response.
```suggestion
				resp, err := clients.Subnet.Get(ctx, resourceGroup, vnetName, subnet, nil)
				g.Expect(err).NotTo(HaveOccurred())
				s := resp.Subnet
```

Reply:
ditto